### DISCUSSION TITLE
How to run storybook?

### DISCUSSION THREAD

**Discussion by alexdev888** [formbricks/formbricks#2629] [General Requests]
Hello community,
I was curious to see how storybook was implemented in this project but was unable to even start storybook. 
When I run **pnpm run storybook** it says:

` ERR_PNPM_NO_SCRIPT  Missing script: storybook`

**Comment by mattinannt** [comment]
@alexdev888 We set up storybook but haven't really used it yet. To run it please move into the storybook folder (`cd apps/storybook`) and run it using pnpm `pnpm dev`
